CHANGELOG.md

```diff
@@ -8,10 +8,10 @@ To install release candidates run `[sudo] gem install cocoapods --pre`
 
 ##### Enhancements
 
-* Remove dependency of `typhoeus`
+* Remove dependency of `typhoeus`.
   [Jun Jiang](https://github.com/jasl)
   [#10346](https://github.com/CocoaPods/CocoaPods/pull/10346)
-
+  
```

Comment:
@jasl I'm sorry I wasn't clear - the two spaces are supposed to be at the end of line 11, not 14 😂